### PR TITLE
Fixed Classic mode: Toolbar: Remove extra separators

### PR DESCRIPTION
**Pull request by codewithvk**
Signed-off-by: vivekpatel7202@gmail.com <vivekpatel7202@gmail.com>
Change-Id: I4844124e2471fc1b9d4b5c008e92fe82ff9418a8

* Resolves: #2071 
* Target version: master 

### Summary

temp

### CODE REVIEW

**Comment by codewithvk** [comment]
@pedropintosilva Can you review it, so I can create new PR accordingly.

**Comment by pedropintosilva** [comment]
I have tested and seems good but it only affects drawing documents, you can also do the same for other apps

**Comment by codewithvk** [comment]
@pedropintosilva I had one mistake with git. By Mistake i --amend my commit with other one's commit. Because I fetch data so last commit was someone else. Can you help me with this?
temp2

**Comment by pedropintosilva** [comment]
Yes, no problem it happens:
- First make sure you save you changes somewhere else (if you have nothing that bigger and you remember what you change skip this step)
- reset your branch `git reset HEAD~1 --hard`
- then `git pull -r`

and now you can do your changes

- `git add .`
- `git commit --signoff`
- Message
- push

------

Next time you have dirty directory (with untracked changes etc) and you want to fetch from upstream you have two options:
1. Save your changes first in a stash `git stash push -m "yourstash"` or `git stash save "yourstash"` (git stash --help); pull; and then you can apply your saved stash above `git stash apply stash@{0} ` (do `git stash list` to know which number it is)
2. Or you can first add your changes to a local commit and then rebase with `git rebase master`

**Comment by codewithvk** [comment]
@pedropintosilva thank you So much for your guidance. 😃

**Comment by codewithvk** [comment]
t1
Should I remove the separator from here?

**Comment by codewithvk** [comment]
@pedropintosilva Can you look it?

**Comment by codewithvk** [comment]
@pedropintosilva Can you review it, So I can work with another issue.

**Review by pedropintosilva — COMMENTED** [review]
sorry for the delay @codewithvk did you test it? because to change these you really need to test with some integrator (for example Nextcloud), as I see it seems that now if those integrator-functionalities are available we are missing important separators. So maybe for now would be better to test in all apps. Hm maybe it would be easier to **first change the order of some items** in Draw so we have everything the same and then worry about the sparators in another PR.

image

and make sure that in Drawing (3rd image) the order is:

1. See revision history
2. Download as
3. Print...
4. Close document

```
		drawing: [
			{name: _UNO('.uno:PickList', 'presentation'), id: 'file', type: 'menu', menu: [
				{name: _UNO('.uno:Save', 'presentation'), id: 'save', type: 'action'},
				{name: _UNO('.uno:SaveAs', 'presentation'), id: 'saveas', type: 'action'},
				{name: _('Save Comments'), id: 'savecomments', type: 'action'},
				{name: _('Share...'), id:'shareas', type: 'action'},
				{name: _UNO('.uno:Print', 'presentation'), id: 'print', type: 'action'},
				{name: _('See revision history'), id: 'rev-history', type: 'action'},
				{name: !window.ThisIsAMobileApp ? _('Download as') : _('Export as'), id: 'downloadas', type: 'menu', menu: [
					{name: _('PDF Document (.pdf)'), id: 'downloadas-pdf', type: 'action'},
					{name: _('ODF Drawing (.odg)'), id: 'downloadas-odg', type: 'action'}
				]},
				{name: _('Close document'), id: 'closedocument', type: 'action'}
			]},
```

**Comment by codewithvk** [comment]
>test with some integrator (for example Nextcloud),

I haven't an idea, How can I test with nextCloud, Can you tell me or you can give some articles. So I can figure out that!

File: loleaflet/src/control/Control.Menubar.js [hunk 1]
```diff
@@ -402,7 +402,6 @@ L.Control.Menubar = L.Control.extend({
 					{name: _('PDF Document (.pdf)'), id: 'downloadas-pdf', type: 'action'},
 					{name: _('ODF Drawing (.odg)'), id: 'downloadas-odg', type: 'action'}
 				]},
-				{type: 'separator'},
```
[the remark this answers is not in the document]
Comment (codewithvk) [reply loleaflet/src/control/Control.Menubar.js:405]: Okey I will do it.

**Comment by codewithvk** [comment]
> and make sure that in Drawing (3rd image) the order is:
> 
> 1. See revision history
> 2. Download as
> 3. Print...
> 4. Close document

I didn't change order, But I will make in order for sure :)

**Comment by codewithvk** [comment]
Also, I Found a small bug, If it is a bug then you can create an issue.
temp
File menu button Override fullscreen Button of the menu.

File: loleaflet/src/control/Control.Menubar.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (codewithvk) [reply loleaflet/src/control/Control.Menubar.js:405]: I think in spreadsheet, text, and presentation don't have separate in `File` of `Menu` section.

https://github.com/CollaboraOnline/online/blob/7f073c7f8804cde93ae648cc89ef96f21fc0c0d0/loleaflet/src/control/Control.Menubar.js#L31
https://github.com/CollaboraOnline/online/blob/7f073c7f8804cde93ae648cc89ef96f21fc0c0d0/loleaflet/src/control/Control.Menubar.js#L277

**Comment by pedropintosilva** [comment]
bug created: https://github.com/CollaboraOnline/online/issues/2165 thanks!

**Comment by codewithvk** [comment]
@pedropintosilva Can you please look at it?

**Comment by pedropintosilva** [comment]
@codewithvk sorry I kinda forgot about this PR. Thanks for ping me. However I think we should just close this PR as it is. Because and as pointed in the description of https://github.com/CollaboraOnline/online/issues/2071 this removal should be conditional: check if the menu items (that required separator before them) are available and only show the separators in that case. 

So I would recommend instead to:

> Hm maybe it would be easier to first change the order of some items in Draw so we have everything the same and then worry about the sparators in another PR.
https://github.com/CollaboraOnline/online/pull/2078#pullrequestreview-642104399

**Pull request closed without merging**